x/qgb/README.md

```diff
@@ -0,0 +1,140 @@
+# Quantum gravity bridge state machine
```

Comment:
[no change needed][discussion] since this README contains docs for a Cosmos SDK module, thoughts on conforming to the structure of https://github.com/cosmos/cosmos-sdk/tree/main/x/auth. For this line:

```suggestion
# `x/qgb`
```

if we do adopt the Cosmos SDK module docs structure, we may also adopt the headings from https://github.com/cosmos/cosmos-sdk/tree/main/x/auth for other sections